app/dashboard/static/js/app/view-jobs-job-branch.2017.4.js

```diff
@@ -358,18 +358,18 @@ require([
          * Wrapper to provide the href.
         **/
         function _renderTestCount(data, type) {
+            return jobt.renderTestCount({data: data, type: type});
+        }
+
+        function _renderBuildCount(data, type) {
             var href = '/test/job/';
```

Comment:
This function is called `_renderBuildCount()`, to show the number of builds with a link to the builds view.

The URL starts with `/test/job/`, which obviously is not correct.

Please, wait for a little while and check your changes locally before pushing to a PR.

Reply:
Sure, I will. 
I updated to `_renderTestJobBranchKernel`, but not sure about the name.

Reply:
The issue was the other way round, the function name is fine but not the URL... This column is to get a link to the builds.

Reply:
I see, so now we have another magnifying to be removed.